fluent/src/resolver.js

```diff
@@ -98,6 +132,11 @@ function MessageReference(env, {name}) {
 /**
  * Resolve an array of tags.
  *
+ * @param   {Object} env
+ *    Resolver environment object.
+ * @param   {String} name
+ *    ID of a message for which the tags are resolved.
```

Comment:
Here again you'll need to account for the fact that `name` is being defined by destructing the second argument to the function.